rllib/utils/minibatch_utils.py

```diff
@@ -87,7 +86,13 @@ def __iter__(self):
                     def get_len(b):
                         return len(b[SampleBatch.SEQ_LENS])
 
+                    n_steps = int(
```

Comment:
Bug fix. When slicing on a BxT batch, we should slice properly along B-axis (with the correct slice size!).